@kiva/kv-components/src/vue/KvWwwHeader/KvHeaderLinkBar.vue

```diff
@@ -19,32 +19,32 @@
 			/>
 		</button>
 		<!-- lend -->
-		<a
-			ref="lendButton"
+		<KvHeaderDropdownLink
 			v-kv-track-event="['TopNav', 'click-Lend']"
-			href="/lend-by-category"
-			class="
-				tw-px-1.5 tw-py-1 tw-mx-1
-				tw-border-2
-				tw-text-eco-green-3 tw-border-current
-				tw-no-underline hover:tw-no-underline
-				tw-inline-flex tw-items-center
-			"
-			:class="{
-				'tw-text-tertiary': openMenuItem && openMenuItem !== KvLendMenu,
-			}"
-			style="border-radius: 1rem;"
-			@mouseover="onHover(lendButton, KvLendMenu)"
-			@mouseout="onHover()"
+			ref-name="lendButton"
+			:href="'/lend-by-category'"
+			:menu-component="KvLendMenu"
+			:open-menu-item="openMenuItem"
+			:on-hover="onHover"
+			:dropdown-icon="mdiChevronDown"
+			base-class="tw-border-2 tw-border-current tw-inline-flex"
 		>
 			Lend
 			<span class="tw-hidden lg:tw-inline tw-ml-0.5">now</span>
-			<kv-material-icon
-				class="tw-hidden md:tw-inline tw-w-3 tw-ml-0.5 tw-transition-transform tw-duration-300"
-				:class="{'tw-rotate-180' : openMenuItem === KvLendMenu}"
-				:icon="mdiChevronDown"
-			/>
-		</a>
+		</KvHeaderDropdownLink>
+		<!-- Take Action -->
+		<KvHeaderDropdownLink
+			v-kv-track-event="['TopNav', 'click-TakeAction']"
+			ref-name="takeActionButton"
+			base-class="tw-hidden lg:tw-inline-flex"
+			:href="'/take-action'"
```

Comment:
I don't think this nav item needs an action on click, it's just opening the menu.